ZAPD/ZBlob.cpp

```diff
@@ -61,24 +58,18 @@ ZBlob* ZBlob::FromFile(const std::string& filePath)
 string ZBlob::GetSourceOutputCode(const std::string& prefix)
 {
 	sourceOutput = "";
-	// sourceOutput += StringHelper::Sprintf("u8 %s_%s[] = \n{\n", prefix.c_str(), name.c_str());
 
 	for (uint32_t i = 0; i < rawData.size(); i += 1)
 	{
 		if (i % 16 == 0)
-			sourceOutput += "\t";
+			sourceOutput += "    ";
 
 		sourceOutput += StringHelper::Sprintf("0x%02X, ", rawData[i]);
 
 		if (i % 16 == 15)
 			sourceOutput += "\n";
 	}
 
-	// Ensure there's always a trailing line feed to prevent dumb warnings.
-	sourceOutput += "\n";
```

Comment:
yea, every source file needs a newline at the end.
We could possibly make this a part of the output formatter, if we end up feeding the whole file to it instead of individual parts. havent really figured out how that should work yet. but this should stay for now